packages/gatsby-cli/src/structured-errors/error-map.js

```diff
@@ -76,6 +76,14 @@ const errorMap = {
     type: `CONFIG`,
     level: `ERROR`,
   },
+  "10126": {
+    text: context =>
+      `${context.pluginName} threw an error while running the ${
+        context.api
+      } lifecycle:\n\n${context.message}`,
+    type: `CONFIG`,
```

Comment:
we might want to convert this to a plugin type
```suggestion
    type: `PLUGIN`,
```